### PR TITLE
Add kahoot

### PR DESCRIPTION
**Pull request by sn00py1310**

### CODE REVIEW

File: entries/k/kahoot.it.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Kahoot!": {
+    "domain": "kahoot.it",
```
Comment (ApCoder123) [inline entries/k/kahoot.it.json]:
```suggestion
    "domain": "kahoot.com",
    "additional-domains": [
      "kahoot.it"
    ],
```
While the quizzes are accessible from `kahoot.it`, the account login, management and other sensitive details (e..g private kahoots) are available at `kahoot.com`. Please also change the file name to `kahoot.com.json`.

I've also suggested adding `kahoot.it` as an additional domain, as there are some login pages that use subdomains of that (e.g. `create.kahoot.it`).

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]
Thanks for the pull request @sn00py1310!

**Comment by sn00py1310** [comment]
@Carlgo11 I appreciate the project and like to do something useful. Otherwise, I would waste my time. And I learn more about Github and Git. 
All my entries are from a simple password manager script to compare the domains.
I need to thank all of you for helping me with my wrong entries and SVGs.

**Comment by Carlgo11** [comment]
@sn00py1310 thank you for the kind words! 😊

File: entries/k/kahoot.com.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "Kahoot!": {
+    "domain": "kahoot.com",
+    "additional-domains": [
+      "kahoot.it"
+    ],
+    "contact": {
+      "facebook": "getkahoot",
+      "twitter": "Kahoot"
```
Comment (ApCoder123) [inline entries/k/kahoot.com.json]:
```suggestion
      "twitter": "KahootSupport"
```

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**